x-pack/plugins/alerting/server/saved_objects/index.ts

```diff
@@ -125,6 +129,13 @@ export function setupSavedObjects(
     mappings: rulesSettingsMappings,
   });
 
+  savedObjects.registerType({
+    name: MAINTENANCE_WINDOW_SAVED_OBJECT_TYPE,
+    hidden: true,
+    namespaceType: 'single',
```

Comment:
I'm not too familiar with alerting and related features and the scope of use cases, but just wanted to make sure we're ok with this being a single namespace type.

Reply:
Yes this is a good point, we don't foresee any reason to make the maintenance window SO accessible across spaces but I will chat with a few more folks on the team just to be sure. Thanks for pointing this out.

Reply:
@JiaweiWu Even if you think you won't use it, assigning a namespace type of `multiple-isolated` is probably worth doing. This will give you the space isolation of `single`, but it will also leave the possibility open to making this sharable at any point in the future, without having to wait for a painful Breaking Change event.

Reply:
Sounds good, I will use `multiple-isolated` in that case :smile: